web3.js/package.json

```diff
@@ -55,6 +55,7 @@
     "re": "semantic-release --repository-url git@github.com:solana-labs/solana-web3.js.git",
     "test": "mocha -r ts-node/register './test/**/*.test.ts'",
     "test:cover": "TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' nyc --reporter=lcov mocha -r ts-node/register './test/**/*.test.ts'",
+    "test:esm-compat": "node test/esm-compat.mjs",
```

Comment:
Perhaps add this invocation to the CI definitions as well?

Reply:
good point, added

Reply:
that's so good ！